parkhangs-webapp/src/components/EventForm.js

```diff
@@ -228,7 +249,7 @@ const mapDispatchToProps = (dispatch) => ({
 
 const mapStateToProps = (state) => ({
     parks: state.parks.parks,
-    user: state.user.user
+    user: state.user.user,
```

Comment:
can remove the comma

Reply:
trailing commas are 👌 leave it please, I've been trying to use them everywhere haha https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8